crates/plugin_lodash/src/lib.rs

```diff
@@ -98,6 +119,7 @@ pub struct PluginLodash {
 
 impl PluginLodash {
   fn add_import(&mut self, source: JsWord, local: Id, imported: Option<Id>) -> Id {
+    let (mappings, pkg_map, _) = &*CACHED.read();
```

Comment:
It will be a better practice if we place the lock and the consuming part together. This keeps us from holding the lock for too much time which may have an impact on the overall performance. In this case, you can do this:
```rust
let new_source = {
      let (mappings, pkg_map, _) = &*CACHED.read();
      let pkg = pkg_map.get(&source).unwrap();
      let import_path = pkg
        .find_module(mappings, &imported_name)
        .unwrap_or_else(|| {
          panic!(
            "Cannot find appropriate import path to: {}, in package: {}",
            imported_name, source
          )
        });
      format!("{}/{}", pkg.id, import_path)
};
```